src/main/java/deku/task/Task.java

```diff
@@ -0,0 +1,116 @@
+package deku.task;
+
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.util.List;
+
+import deku.DekuExceptions;
+import deku.InputParser;
+
+/**
+ * Main parent Task class
+ */
+public class Task {
+    private final List<String> taskArray;
+    private final String task;
+    private final String icon;
+    private Boolean completionStatus;
+    private String completionIcon;
+    private final InputParser parser = new InputParser();

```

Comment:
Perhaps the final variables should be in all capital letters to represent constants?